client/packages/android/package.json

```diff
@@ -18,7 +18,7 @@
   "scripts": {
     "build:server": "./build_remote_server_libs.sh",
     "build:debug": "yarn build:server && ./gradlew assembleDebug",
-    "build:release": "yarn build:server && ./gradlew assembleRelease",
+    "build:release": "yarn build:server && ./gradlew assembleArm64Release assembleUniversalRelease",
```

Comment:
Wow this is super cool. Impressive.
Since there is no direct way to search for assembleArm64Release globally (and the other one), can you please add a comment: "//": "assembleArm64Release and assembleUniversalRelease refers to arm64 and universal in client/packages/android/app/build.gradle"